infra/main.tf

```diff
@@ -0,0 +1,106 @@
+terraform {
+    required_providers {
+        google = {
+            source = "hashicorp/google"
+            version = "4.37.0"
+        }
+    }
+}
+
+variable "vm_name" {
+    type = string
+}
+
+variable "user" {
+    type = string
+}
+
+variable "credentials_file" {
+    type = string
+}
+
+provider "google" {
+    credentials = file("${var.credentials_file}")
+
+    project = "textile-310716"
+    region  = "us-west1"
+    zone    = "us-west1-b"
+}
+
+resource "google_compute_instance" "validator" {
+    name         = "${var.vm_name}"
+    machine_type = "e2-standard-2"
+    tags         = ["https-server", "grafana", "ssh-vm"]
+    deletion_protection = false
```

Comment:
In our infra docs we suggest setting this to `true`. 
No strong opinion in doing the same here, just checking if was intentional.